src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,67 @@
+import calculate from '../logic/calculate';
+
+describe('calculate', () => {
+  it('should return an object', () => {
+    expect(typeof calculate({}, 'AC')).toBe('object');
+  });
+
+  // it('should return an object with total, next and operation properties', () => {
+  //   expect(calculate({}, 'AC')).toEqual({
+  //     total: null,
+  //     next: null,
+  //     operation: null,
+  //   });
+  // });
+
+  it('should add two numbers', () => {
+    const obj = { total: '1', next: '2', operation: '+' };
+    const expected = { total: '3', next: null, operation: null };
+
+    expect(calculate(obj, '=')).toEqual(expected);
+  });
+
+  it('should subtract two numbers', () => {
+    const obj = { total: '1', next: '2', operation: '-' };
+    const expected = { total: '-1', next: null, operation: null };
+
+    expect(calculate(obj, '=')).toEqual(expected);
+  });
+
+  it('should multiply two numbers', () => {
+    const obj = { total: '1', next: '-2', operation: 'x' };
+    const expected = { total: '-2', next: null, operation: null };
+
+    expect(calculate(obj, '=')).toEqual(expected);
```

Comment:
- [ ] Nice approach :+1:

Reply:
Thanks for the feedback. I really appreciate